### ISSUE TITLE
Add repository to HACS

### ISSUE THREAD

**Issue by MadOne** [OStrama/weishaupt_modbus#42]
**Is your feature request related to a problem? Please describe.**
Everyone who wants to use our integration has to find it and add it to hacs,

**Describe the solution you'd like**
Add our integration to hacs so everywone using hacs can find it

**Describe alternatives you've considered**
Leave it as is.

**Additional context**

All files required for the integration to run must be located inside the directory ROOT_OF_THE_REPO/custom_components/INTEGRATION_NAME/.

The kennfeld Files are NOT. Maybe all "kennfelder" in one file? A dropdown?

update info.md

https://hacs.xyz/docs/publish/start/#my-links
https://hacs.xyz/docs/publish/integration/

**Comment by OStrama** [comment]
All in one file would it make more difficult for users to create new files..

**Comment by MadOne** [comment]
I agree.
Maybe a template.json for local edit and send to us and one file where we define the known curves.
Dropdown with known curves and custom for the template.json.

But for now this is perfectly fine. We fullfill the requirements for beeing listed in hacs.

Do create a pull request? (You are the owner)

**Comment by OStrama** [comment]
Yes, will do. But what version should we use?

**Comment by MadOne** [comment]
The latest stable (1.0.1). Then we should wait pushing ne versions until we got merged.

**Comment by MadOne** [comment]
I see you pushed the file moves to the translation branch. Should we wait for the translaiton to finish, merge to main and publish 1.0.2 before we push to hacs?

**Comment by OStrama** [comment]
No, I can transfer them to version-1.0 and introduce it in HACS. But I have to debug finding always the right config dir before..

**Comment by OStrama** [comment]
Version 1.0.2 is out, temporarily without translation, because there was an issue that I found quite late on my production system:

Since we had to set _attr_has_entity_name = True, the names of the entities now contain the device name, too. This breaks statistics.

We have to find a solution for 1.0.3, either overloadeing _friendly_name_internal() or let the names change and do something like this as a repair step:
https://community.home-assistant.io/t/merge-statistics-of-old-entity-to-new-entity/467816/42

**Comment by MadOne** [comment]
### **https://github.com/hacs/default/pull/2855**

for easier finding.

**Comment by MadOne** [comment]
![image](https://github.com/user-attachments/assets/f131faac-9c9c-4c02-a5bb-390c021aa4c3)

This is new, istnt it?

**Comment by MadOne** [comment]
@OStrama ?

**Comment by OStrama** [comment]
I had a look at the conflict. It's not related to our integration:

  "zigul/HomeAssistant-CEZdistribuce",
<<<<<<< patch-1
  "ZsBT/hass-w1000-portal",
  "zweckj/acaia"
=======
  "ZsBT/hass-w1000-portal"
>>>>>>> master
]

I have no write access to solve it..

**Comment by OStrama** [comment]
De-done pull request: https://github.com/hacs/default/pull/3015

**Comment by OStrama** [comment]
It takes a while to add the integration to HACS. Here's the current queue: https://github.com/hacs/default/pulls?q=is%3Apr+is%3Aopen+draft%3Afalse+sort%3Acreated-asc

**Comment by OStrama** [comment]
We are part of HACS now!

https://github.com/hacs/default/pull/3015
